tests/test_forms.py

```diff
@@ -25,3 +28,39 @@ def test_only_ons_email_address_valid(self):
                 result = form.validate()
                 self.assertTrue(result)
                 self.assertEqual(len(form.errors), 0)
+
+    def test_valid_password_for_activation_and_reset(self):
```

Comment:
Minor, but this should be more generic, it should be something like test_set_account_password, where we use it is not relevant and plays no part in the test. I usually drop the valid bit on successful tests otherwise we would end up with it on everything that should work, only putting it on invalid ones